packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -20,8 +21,8 @@ class Compiler {
                 process.stdout.write('\n');
                 const defaultProgressPluginHandler = (percent, msg) => {
                     percent = Math.floor(percent * 100);
-                    process.stdout.clearLine();
-                    process.stdout.cursorTo(0);
+                    readline.clearLine(process.stdout, 0);
+                    readline.cursorTo(process.stdout, 0, null);
```

Comment:
Why?

Reply:
Because when running with jest test was failing with error `TypeError: process.stdout.clearLine is not a function`. 

Maybe because `clearLine` and `cursorTo` are undefined when `process.stdout` is not a tty

More info => https://stackoverflow.com/questions/34570452/node-js-stdout-clearline-and-cursorto-functions